### PR TITLE
cli upgrades (appInfo, appPort, & adminPort opts)

### PR DESCRIPTION
**Pull request by JettTech**
### Updates: 
 - re-formats cli to use commander
 - adds app and admin port options
  - fixes state dump arg issue
 - combines prs #5 and #6 with some cleanup
 - bumps holochain-conductor-api to v0.0.4

### CODE REVIEW

**Review by zippy — CHANGES_REQUESTED** [review]
I tested appInfo and it looks good.
However when I tried -s (dumpState) I noticed that that fails now, and it would be good to have that working too while we are doing clean up.  Can you look into that?

**Review by peeech — CHANGES_REQUESTED** [review]
1. If you look at github there's a lot of whitespace formatting issues. Might be nice to stick to standard.js formatting.
2. README.md should probably be updated
3. It might be nice to catch() errors produced by async function `call_admin_port()`, otherwise you get `Unhandled promise rejections`

**Review by zippy — APPROVED** [review]
Woot!

File: README.md [hunk 1]
```diff
@@ -3,35 +3,45 @@
...
-            or numeric index of cell ID returned by ListCellIds
-
-    example
-        hc-state -s "[
-            'hC0kqcfqvJ8krBR0bNnPsmLtFiEiMOHM0fX+U8FW+ROc7P10tUdc',
-            'hCAkcIRv7RZNVg8FWc6/oJZo04dZTXm7JP6tfMk3RptPY02cBQac'
-        ]"
+            
+        example
+            hc-state s "['hC0kqcfqvJ8krBR0bNnPsmLtFiEiMOHM0fX+U8FW+ROc7P10tUdc','hCAkcIRv7RZNVg8FWc6/oJZo04dZTXm7JP6tfMk3RptPY02cBQac']
```
Comment (zippy) [inline README.md]:
```suggestion
            hc-state s "['hC0kqcfqvJ8krBR0bNnPsmLtFiEiMOHM0fX+U8FW+ROc7P10tUdc','hCAkcIRv7RZNVg8FWc6/oJZo04dZTXm7JP6tfMk3RptPY02cBQac']"
```

**Pull request merged**